documentation/bolt_installing.md

```diff
@@ -85,7 +85,7 @@ choco uninstall puppet-bolt
 PowerShell versions 2.0 and 3.0 cannot automatically discover and load the Bolt
 module, so you'll need to add it manually. Unless your system dates from 2013 or
 earlier, this situation probably does not apply to you. To confirm your version,
-run `echo $PSTableVersion` in PowerShell.
+run `echo $PSVersionTable` in PowerShell.
```

Comment:
```suggestion
run `echo $PSVersionTable` in PowerShell.
```

It looks like you don't need to actually echo `$PSVersionTable` since it's an automatic variable that will print the info on its own.